src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -2092,19 +2092,20 @@ protected function fetchUserNodeTypes()
 
         $query = '
 SELECT 
-phpcr_type_nodes.name AS nodeName, phpcr_type_nodes.is_abstract AS nodeAbstract, 
-phpcr_type_nodes.is_mixin AS nodeMixin, phpcr_type_nodes.queryable AS nodeQueryable, 
-phpcr_type_nodes.orderable_child_nodes AS nodeHasOrderableChildNodes, 
-phpcr_type_nodes.primary_item AS nodePrimaryItemName, phpcr_type_nodes.supertypes AS declaredSuperTypeNames, 
-phpcr_type_props.name AS propertyName, phpcr_type_props.auto_created AS propertyAutoCreated, 
-phpcr_type_props.mandatory AS propertyMandatory, phpcr_type_props.protected AS propertyProtected, 
-phpcr_type_props.on_parent_version AS propertyOnParentVersion, phpcr_type_props.required_type AS propertyRequiredType, 
-phpcr_type_props.multiple AS propertyMultiple, phpcr_type_props.fulltext_searchable AS propertyFulltextSearchable, 
-phpcr_type_props.query_orderable AS propertyQueryOrderable, phpcr_type_props.default_value as propertyDefaultValue,
-phpcr_type_childs.name AS childName, phpcr_type_childs.auto_created AS childAutoCreated, 
-phpcr_type_childs.mandatory AS childMandatory, phpcr_type_childs.protected AS childProtected, 
-phpcr_type_childs.on_parent_version AS childOnParentVersion, phpcr_type_childs.default_type AS childDefaultType, 
-phpcr_type_childs.primary_types AS childPrimaryTypes 
+phpcr_type_nodes.name AS node_name, phpcr_type_nodes.is_abstract AS node_abstract, 
```

Comment:
why does this show up in this PR? did we not already merge this? btw, it is a good idea to create branches for each pull request in your repository, so that you can update each thing individually.

can you rebase on master so that we see if the node type query commit goes away from the diff? or rebase this branch and remove the duplicated commit?